### PR TITLE
Ui navigation

### PR DESCRIPTION
**Pull request by f11xter**
Emulator not working so couldn't test
Should work fine

### CODE REVIEW

File: mapping_project/lib/custom_app_bar.dart [hunk 1]
```diff
@@ -1,39 +1,24 @@
...
-import 'settings_page.dart';
-import 'lessons_page.dart';
-import 'directionsSummaryPage.dart';
-import 'myhomepage.dart';
-import 'tannoy.dart';
 
 class CustomAppBar {
   static void _pushSettingsPage(BuildContext context) {
-    Navigator.of(context).pushReplacement(
-      MaterialPageRoute(builder: (BuildContext context) => MyApp()),
```
Comment (zst-c) [inline mapping_project/lib/custom_app_bar.dart:11]: Do we need this still - I assume we can get rid of the lines: 

`Navigator.of(context).pushReplacement(
      MaterialPageRoute(builder: (BuildContext context) => MyApp()),);`

As we have the pushReplacementNamed instead

**Review by zst-c — CHANGES_REQUESTED** [review]
Rest seems good except that comment I left - will merge after

File: mapping_project/lib/custom_app_bar.dart (hunk 1 above)
Comment (f11xter) [reply mapping_project/lib/custom_app_bar.dart:11]: I think I've already done that - you might have commented on the old code?

Comment (zst-c) [reply mapping_project/lib/custom_app_bar.dart:11]: ahhh yes - GitHub didn't show me that this morning!

**Review by zst-c — APPROVED** [review]
All good

**Pull request merged**